src/logger/src/metrics.rs

```diff
@@ -626,6 +630,19 @@ pub struct RTCDeviceMetrics {
     pub missed_write_count: SharedIncMetric,
 }
 
+#[cfg(target_arch = "aarch64")]
+impl RTCEvents for RTCDeviceMetrics {
+    fn invalid_read(&self) {
+        self.missed_read_count.inc();
+        warn!("Invalid read from RTC device.")
```

Comment:
I would make the error specific: `Guest read/write at invalid offset`. For debug purposes it would help to also have the offset here.